### PR TITLE
Sync with internal repo - tuned rules

### PR DESCRIPTION
**Pull request by nbeckstead-ddog**
### What does this PR do?

Updated Detections
* `aws_metadata_service`
* `critical_binaries` FIM
* `credential_modified` FIM

### Motivation

Tuning rules.

### CODE REVIEW

**Comment by paulcacheux** [comment]
You can ignore one of the error in the CI since it's related to new splice events that are not in the last stable release

File: runtime/default.policy [hunk 1]
```diff
@@ -104,6 +104,8 @@ rules:
           (chmod.file.path in [ "/etc/shadow", "/etc/gshadow" ])
           && process.file.path not in [ "/sbin/vipw", "/usr/sbin/vipw", "/sbin/vigr", "/usr/sbin/vigr" ]
           && process.ancestors.file.path not in SYSTEM_PACKAGE_MANAGERS
+          && (process.file.name not in ["containerd"] && process.args in ["info"])
```
Comment (paulcacheux) [inline runtime/default.policy:107]:
```suggestion
          && !(process.file.name in ["containerd"] && process.args in ["info"])
```

**Review by paulcacheux — CHANGES_REQUESTED** [review]

File: runtime/default.policy (hunk 1 above)
Comment (paulcacheux) [reply runtime/default.policy:107]: I think the boolean logic is wrong here (and in all similar rules). Did this rule work when you tested it locally ?

Comment (nbeckstead-ddog) [reply runtime/default.policy:107]: You're right. Fixed in https://github.com/DataDog/security-monitoring/pull/938

**Pull request closed without merging**